src/plugins/intel_cpu/src/nodes/executors/x64/subgraph.hpp

```diff
@@ -22,20 +23,29 @@ class SubgraphExecutor : public SubgraphBaseExecutor {
                  const std::vector<MemoryPtr>& inMemPtrs,
                  const std::vector<MemoryPtr>& outMemPtrs) override;
 
+    static std::vector<MemoryPtr> prepareWeights(const std::vector<MemoryPtr>& inMemPtrs,
```

Comment:
```suggestion
    static std::vector<MemoryPtr> prepare_weights(const std::vector<MemoryPtr>& inMemPtrs,
```
To align with other methods names inside the classes